org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/tracking/IssueTrackable.java

```diff
@@ -51,7 +51,7 @@ public Integer getTextRangeHash() {
 
   @Override
   public Integer getLineHash() {
-    // TODO Auto-generated method stub
+    // TODO implement using same algorithm as used by SonarQube
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)